lib/slax/projects/project_repos.ex

```diff
@@ -42,6 +42,7 @@ defmodule Slax.ProjectRepos do
     |> Repo.transaction()
   end
 
+  # sobelow_skip ["DOS.BinToAtom"]
```

Comment:
There probably isn't a serious vulnerability here, but it's always nice to consider the "skip" comment as a last resort. In this case, that second argument in `Multi.update()` [can actually be any elixir term](https://hexdocs.pm/ecto/Ecto.Multi.html#t:name/0), so making it an atom is completely unnecessary. We could just use `"project_repo_#{id}"` without the colon